ground-core/scripts/cassandra/cassandra.sql

```diff
@@ -1,134 +1,136 @@
 /* VERSIONS */
 
-create table Versions (id varchar primary key);
+create table version (
+    id varchar PRIMARY KEY
+);
 
-create table VersionSuccessors (
-    successor_id varchar primary key,
-    vfrom varchar, 
-    vto varchar
+create table version_successor (
+    id varchar PRIMARY KEY,
+    from_version_id varchar,
+    to_version_id varchar
 );
 
-create table Items (
-    id varchar primary key
+create table item (
+    id varchar PRIMARY KEY
 );
 
-create table VersionHistoryDAGs (
+create table version_history_dag (
     item_id varchar,
-    successor_id varchar,
-    primary key(item_id, successor_id)
+    version_successor_id varchar,
+    PRIMARY KEY(item_id, version_successor_id)
 );
 
 /* MODELS */
 
-create table Structures (
+create table structure (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
```

Comment:
What do you mean by more than one name? `id` is an internal key, and `name` is an external facing name that has to be unique.